testdata/ingress_rg_notshared_alb/template.cf

```diff
@@ -0,0 +1,219 @@
+        },
+        "TargetGroupHealthCheckPathParameter": {
+            "Type": "String",
+            "Default": "/kube-system/healthz",
+            "Description": "The healthcheck path"
+        },
+        "TargetGroupHealthCheckPortParameter": {
+            "Type": "Number",
+            "Default": "9999",
+            "Description": "The healthcheck port"
+        },
+        "TargetGroupHealthCheckTimeoutParameter": {
+            "Type": "Number",
+            "Default": "5",
+            "Description": "The healthcheck timeout"
+        },
+        "TargetGroupTargetPortParameter": {
+            "Type": "Number",
+            "Default": "9999",
+            "Description": "The target port"
+        },
+        "TargetGroupVPCIDParameter": {
+            "Type": "AWS::EC2::VPC::Id",
+            "Description": "The VPCID for the TargetGroup"
+        },
+        "Type": {
+            "Type": "String",
+            "Default": "application",
+            "Description": "Loadbalancer Type, 'application' or 'network'"
+        }
+    },
+    "Resources": {
+        "HTTPListener": {
+            "Type": "AWS::ElasticLoadBalancingV2::Listener",
+            "Properties": {
+                "DefaultActions": [
+                    {
+                        "TargetGroupArn": {
+                            "Ref": "TG"
+                        },
+                        "Type": "forward"
+                    }
+                ],
+                "LoadBalancerArn": {
+                    "Ref": "LB"
+                },
+                "Port": 80,
+                "Protocol": "HTTP"
+            }
+        },
+        "HTTPSListener": {
+            "Type": "AWS::ElasticLoadBalancingV2::Listener",
+            "Properties": {
+                "Certificates": [
+                    {
+                        "CertificateArn": "DUMMY"
+                    }
+                ],
+                "DefaultActions": [
+                    {
+                        "TargetGroupArn": {
+                            "Ref": "TG"
+                        },
+                        "Type": "forward"
+                    }
+                ],
+                "LoadBalancerArn": {
+                    "Ref": "LB"
+                },
+                "Port": 443,
+                "Protocol": "HTTPS",
+                "SslPolicy": {
+                    "Ref": "ListenerSslPolicyParameter"
+                }
+            }
+        },
+        "HTTPSListenerCertificatefc48082457b770e278fc0bd3d392d127869993166f76e8df57d19a0e662820ea": {
+            "Type": "AWS::ElasticLoadBalancingV2::ListenerCertificate",
+            "Properties": {
+                "Certificates": [
+                    {
+                        "CertificateArn": "DUMMY"
+                    }
+                ],
+                "ListenerArn": {
+                    "Ref": "HTTPSListener"
+                }
+            }
+        },
+        "LB": {
```

Comment:
This template is exactly the same as `ingress_rg_shared_alb`, this is not what I expected. Since there is an ingress and route_group configured and they should not share the same LB I would expect two LBs in the template. Did I get something wrong?

Reply:
This comment from Mustafa seems to be the answer to my question: https://github.com/zalando-incubator/kube-ingress-aws-controller/pull/614#discussion_r1229775130

Reply:
The only weird thing is that if it creates two stacks, this stacks should have different tags, one LB should not be shared and the other LB should be sharable. From what I understood this difference should be reflected by the tags, something like `"ingress:owner":"$RGName"` in the LB/stack that is not sharable.

This means I would have to identify which stack is from which k8s object and see if the outputs match with the expected response. I still don't know how to do that.